Content.Server/LandMines/LandMineSystem.cs

```diff
@@ -14,6 +15,7 @@ public sealed class LandMineSystem : EntitySystem
 
     public override void Initialize()
     {
+        base.Initialize();
```

Comment:
We like padding here.

```suggestion
        base.Initialize();
        
```